WcaOnRails/spec/controllers/persons_controller_spec.rb

```diff
@@ -21,7 +21,7 @@
         json_person = json['rows'][0]
         expect(json_person['name']).to include "Jennifer Lawrence"
         expect(json_person['wca_id']).to eq "2016LAWR01"
-        expect(json_person['country']).to eq "USA"
+        expect(json_person['country']).to eq "United States"
```

Comment:
This is part of the continent/event/country translations, and it's probably the most annoying change...
Translations are provided by the gem [i18n-country-translations](https://github.com/onomojo/i18n-country-translations), and there is a few countries having another name than the one we have now.
One of them is the USA, other ones are Macau (Macau SAR China), and Hong Kong (Hong Kong SAR China). I did not check all countries but I noticed at least these.

I tried a simple overriding in our own locale file but it doesn't work. I'm pretty sure we could find a solution if needed, but before digging more I'd like to know how annoying this is and if it's really needed :p